### PR TITLE
AWS DynamoDB node refactored

### PR DESCRIPTION
**Pull request by ivov**
This PR refactors and expands on the DynamoDB node created in #1581.

### CODE REVIEW

File: packages/nodes-base/nodes/Aws/DynamoDB/AwsDynamoDB.node.ts [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+				// ----------------------------------
+
+				// https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Scan.html
+
+				const eavUi = this.getNodeParameter('expressionAttributeValues.details', i) as IAttributeValueUi[];
+
+				const body: IRequestBody = {
+					TableName: this.getNodeParameter('tableName', i) as string,
+					FilterExpression: this.getNodeParameter('filterExpression', i) as string,
+					ExpressionAttributeValues: adjustExpressionAttributeValues(eavUi),
```
Comment (mvhirsch) [inline packages/nodes-base/nodes/Aws/DynamoDB/AwsDynamoDB.node.ts:283]: ExpressionAttributeValues on "scan" operation is optional. 

Just wanted to add a bugfix to my branch, but you somehow already refactored it. :sweat_smile:

**Comment by mvhirsch** [comment]
@RicardoE105 any chance to get this merged?

**Comment by RicardoE105** [comment]
@mvhirsch it's almost done. Just have to make some changes to the create/update operation. Hopefully will be added to the next released. But not completely sure.

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with n8n@0.130.0

**Pull request merged**